TrainingCenterCRM.WebApi/Controllers/TeachersController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TrainingCenterCRM.BLL.DTO;
+using TrainingCenterCRM.BLL.Interfaces;
+using TrainingCenterCRM.DAL.Enttities;
+
+namespace TrainingCenterCRM.WebApi.Controllers
+{
+    [Route("api/teachers")]
+    [ApiController]
+    public class TeachersController : ControllerBase
+    {
+        private readonly ITeacherService teacherService;
+
+        public TeachersController(ITeacherService teacherService)
+        {
+            this.teacherService = teacherService;
+        }
+        [HttpGet]
+        public IEnumerable<Teacher> GetTeachers()
+        {
+            return teacherService.GetTeachers();
+        }
+
+        [HttpGet("{id}")]
+        public TeacherDTO GetTeacher(int id)
```

Comment:
Which rule is applied to decide which type shild be returned - Teacher or TeacherDTO?